doc/source/actions.rst

```diff
@@ -254,6 +254,17 @@ The ``merge`` action merges the pull request into its base branch. The
        * GPG signed commits will lost their signatures.
        * Also see: :ref:`faq strict rebase`
 
+When the Pull Request is merged, the default merge/squash commit message is the
+Github one. But you can override it by adding a new section into the Pull
+Request body starting with the Markdown title "Commit message", followed by the
+commit message, for example::
+
+    ## Commit message
```

Comment:
I'd suggest `Commit Message` because title must be capitalized in English
(see https://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html)